homeassistant/components/nice_go/strings.json

```diff
@@ -6,12 +6,20 @@
         "data": {
           "email": "[%key:common::config_flow::data::email%]",
           "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "email": "The email address used to log in to the Nice G.O. app",
```

Comment:
My impression is you can reference other strings like `[%key:component::nice_go::config::step::user::data_description::email%]`

Similar to https://github.com/home-assistant/core/blob/8feb6c7e061af9dedbfb6c8e1b340bd2ac855ddc/homeassistant/components/sql/strings.json#L52